packages/file_selector/file_selector_android/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.5.1+8
+
+* Updates example app to target SDK 34.
```

Comment:
I may be wrong about this, but I don't think we need version changes for example app updates @stuartmorgan can you confirm?

Reply:
If we get confirmation, I'll update that. In addition, if we do not need version updates, do we also not need CHANGELOG updates?

Reply:
We definitely don't need version bumps for this; the CI knows not to flag for that. Since this is a routine change that's not interesting to clients, it doesn't need a changelog either. I *think* CI will exempt example app build files, but if the CI check complains you can add the label to override the changelog check and re-run that check.

Reply:
I'm not sure why right now, but it seems that only flutter_plugin_android_lifecycle is complaining about not having a CHANGELOG change. What/where are you referring to adding an override label?

Reply:
He is referring to a github label for the pr named "override: no changelog needed".

Reply:
See for https://github.com/flutter/packages/blob/91caa7a6e3dda76dc0942e87a1ea93409bb3ca16/script/tool/lib/src/version_check_command.dart#L571 how the label is enforced in code.

Reply:
@stuartmorgan Do you have any concerns with this PR as it is now?